src/main/java/com/github/sgov/server/service/repository/VocabularyRepositoryService.java

```diff
@@ -364,6 +364,6 @@ private void removeAllAttachments(VocabularyContext vocabularyContext) {
     }
 
     private void clearApplicationContexts(VocabularyContext vocabularyContext) {
```

Comment:
Little strange that calling public method from private one. Do we even need this method?

Reply:
That's right we don't need it